gulpfile.js

```diff
@@ -2,7 +2,7 @@ var gulp = require('gulp')
 var sass = require('gulp-sass')
 
 gulp.task('sass', function() {
-    return gulp.src('app/scss/**/*.scss') // Gets all files ending with .scss in app/scss
+    return gulp.src(['app/scss/styles.scss', 'app/scss/questionPage.scss', 'app/scss/resultPage.scss']) // Gets all files ending with .scss in app/scss
```

Comment:
I can see why you've done this but there must be a better solution. Every time you want to add a new .scss file you have to go back and edit this, and it would become increasingly longwinded as you add more files.

Reply:
Agree but this is a bit of a legacy issue because scss was not imported originally.  However  there should be no need to add anymore files - all new scss should be inside the  imports folder and imported into one of these three existing files.